### PR TITLE
Magento_Cookie: avoid using deprecated escape* methods from AbstractB…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml [hunk 1]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Cookie\Block\Html\Notices $block */
+/**
+ * @var \Magento\Cookie\Block\Html\Notices $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml]:
```suggestion
/**
 * @var \Magento\Cookie\Block\Html\Notices $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests, Unit Tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Please fix the failing unit tests

**Comment by sergeynezbritskiy** [comment]
> Please fix the failing unit tests

fixed

File: app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cookie/view/frontend/templates/html/notices.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8776](https://jira.corp.magento.com/browse/ENGCOM-8776) has been created to process this Pull Request

**Comment by ihor-sviziev** [comment]
@magento run Database Compare

**Pull request closed without merging**